TW-1102 Postbot section (#4922)
postmanlabs/postman-docs: src/pages/docs/postman-flows/flows-query-language/introduction-to-fql.md

* [Returning structured data](/docs/postman-flows/flows-query-language/returning-structured-results/)
* [Data manipulation](/docs/postman-flows/flows-query-language/data-manipulation/)

## Using Postbot with FQL

> Not sure how to write an FQL statement for your flow? Ask Postbot! Flows has support for artificial intelligence in FQL queries. You can ask Postbot a simple text question, and it will generate a FQL statement to solve your question.

Reviewer: ```suggestion
Flows provides AI assistance with FQL queries. You can ask Postbot a question, and it generates an FQL query.
```

<!--TODO: Image -->

> In the **Evaluate** block, select the Postbot icon <img alt="Postbot icon" src="https://assets.postman.com/postman-docs/v10/icon-postbot-v10-16.jpg#icon" width="18px">. Enter enter your query as a simple text sentence and select **Generate FQL**. Postbot will suggest a query for you.

Reviewer 2: ```suggestion
In the **Evaluate** block, select the Postbot icon <img alt="Postbot icon" src="https://assets.postman.com/postman-docs/v10/icon-postbot-v10-16.jpg#icon" width="18px">. Enter your query as a simple text sentence and select **Generate FQL**. Postbot will suggest a query for you based on your prompt.
```

> In the **Evaluate** block, select the Postbot icon <img alt="Postbot icon" src="https://assets.postman.com/postman-docs/v10/icon-postbot-v10-16.jpg#icon" width="18px">. Enter enter your query as a simple text sentence and select **Generate FQL**. Postbot will suggest a query for you.

Reviewer: ```suggestion
In the **Evaluate** block, select the Postbot icon <img alt="Postbot icon" src="https://assets.postman.com/postman-docs/v10/icon-postbot-v10-16.jpg#icon" width="18px">. Enter your query as a simple sentence, for example, `Check if a field has a specific value`, and select **Generate FQL**. Postbot suggests a query for you.
```

For more information on Postbot, visit [About Postbot](/docs/getting-started/about-postbot/).

## FQL reference

> For a complete reference for all FQL functions, see the [FQL function reference](/docs/postman-flows/flows-query-language/function-reference/).

Reviewer: ```suggestion
For more detailed information, see the [FQL function reference](/docs/postman-flows/flows-query-language/function-reference/).
```
